scripts/workflow.treering.R

```diff
@@ -24,31 +26,38 @@ library(PEcAn.visualization)
 
 #---------------- Load PEcAn settings file. -------------------------------------------------------#
 # Open and read in settings file for PEcAn run.
-settings <- read.settings("~/demo.sda/demo.xml")
+settings <- read.settings("/fs/data2/output//PEcAn_1000001249/pecan.xml")
 #--------------------------------------------------------------------------------------------------#
 
 #---------------- Load plot and tree ring data. -------------------------------------------------------#
 status.start("LOAD DATA")
 ## Read tree data
-trees <- read.csv("/home/carya/Camp2014/ForestPlots/treecores2014.csv")
+trees <- read.csv("/home/araiho/Camp2014/ForestPlots/treecores2014.csv")
 
 ## Read tree ring data
-rings <- Read_Tuscon("/home/carya/Camp2014/ForestPlots/Tucson/")
+source("/home/araiho/Camp2014/statsR/Read_Tuscon.R")
```

Comment:
@araiho did these functions not load via require(PEcAn.data.land)? I know I just updated the exports in that package and so there's a good chance this works now without all these _source_ calls